### PR TITLE
[move-tutorial] address tutorial comments for steps 0, 3, 4 and 6

### PR DESCRIPTION
**Pull request by emmazzz**
## Motivation

This PR makes some changes to the move tutorial:
- Remove mentions of “hackathon” and rename folder to just “tutorial"
- Remove step_0 folder and call the diem dev_setup script directly in step 0
- Inlcude more detailed steps in step 0 like cloning diem repo, `cd`ing into correct directory, etc.
- Remove mentions of `public(script)` from code everywhere
- For steps 3,4,6: move lengthy or advanced contents (like script funs, phantom, Eth comparisons) to dropdowns that are hidden by default
- Address other misc comments

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

Yes.

### CODE REVIEW

File: language/documentation/tutorial/README.md [hunk 1]
```diff
@@ -1,28 +1,55 @@
...
+
+
+- Open up your terminal of choice and clone [`diem`](https://github.com/diem/diem) repo if you don't already have it:
 ```bash
-$ export DIEM_HOME=<path_to_diem_repo>
+$ git clone https://github.com/diem/diem.git
 ```
-- Run the setup script to install Move CLI, Shuffle and dependencies:
 
+- Go to `diem` folder and run the `dev_setup.sh` script:
```
Comment (tzakian) [inline language/documentation/tutorial/README.md:32]: Uber nit: 
```suggestion
- Go to the `diem` folder and run the `dev_setup.sh` script:
```

**Review by tzakian — APPROVED** [review]
Looks great! super minor nit, I'm also happy to fix it in my upcoming PR as well.

**Comment by bmwill** [comment]
looks like GHA had a weird failure i'll restart it
/cancel

**Pull request merged**